assets/styles/style.css

```diff
@@ -0,0 +1,49 @@
+* {
+  background-color: lightgray;
+}
+
+.full {
+  font-size: 25px;
+  font-family: "Times New Roman", Times, serif;
+  text-align: center;
+}
```

Comment:
Notice that there are margins top, right, buttom and left appearing before the main form which contains the tribute image, background color and text on the original page. Make sure that your page resembles that and also review your fonts (family, size, and weight) please.